perf/src/bin/perf_client.rs

```diff
@@ -80,14 +80,7 @@ async fn run(opt: Opt) -> Result<()> {
     info!("connecting to {} at {}", host_name, addr);
 
     let socket = bind_socket(
-        SocketAddr::new(
-            if addr.is_ipv4() {
-                IpAddr::V4(Ipv4Addr::UNSPECIFIED)
-            } else {
-                IpAddr::V6(Ipv6Addr::UNSPECIFIED)
-            },
-            0,
-        ),
+        SocketAddr::new(IpAddr::V6(Ipv6Addr::UNSPECIFIED), 0),
```

Comment:
I feel like it might be more robust to keep this change, since the peer may not be a quinn perf server, and may therefore not be using a dual stack socket.

Reply:
How does it matter for the peer? The stack will always create IPv4 packets. Without the option we are just not allowed to create them based off an IPv6 socket.

Reply:
We can keep the change, but it's then more of a tool to explicitely test IPv4-only endpoints on client side.

Reply:
Oh, I forgot the change affected the client as well. Disregard.